lib/include/up-client-zenoh-cpp/client/upZenohClient.h

```diff
@@ -44,7 +46,9 @@ namespace uprotocol::client {
             * The API provides an instance of the zenoh session
             * @return instance of UpZenohClient
             */
-            static std::shared_ptr<UpZenohClient> instance(void) noexcept;
+            static std::shared_ptr<UpZenohClient> instance(
+                std::optional<uprotocol::v1::UAuthority> = uprotocol::v1::UAuthority(),
+                std::optional<uprotocol::v1::UEntity> = uprotocol::v1::UEntity()) noexcept;
```

Comment:
In this latest update, you've removed `static std::shared_ptr<UpZenohClient> instance () noexcept`. That will increase burden on users of this. The ask was simply to have that call this version with the optional parameters set to `std::nullopt_t`.

You can see the fallout of removing the interface in the scale of changes that were needed in all of the test files.

Additionally, the default parameters here are incorrect. _If_ using default parameters here, the should be `std::nullopt_t`.

Please apply one of these suggestions and roll back to the previous method for retrieving the instances in all of the tests:

```suggestion
            static std::shared_ptr<UpZenohClient> instance() noexcept {
                return instance(std::nullopt_t, std::nullopt_t);
            }

            static std::shared_ptr<UpZenohClient> instance(
                std::optional<uprotocol::v1::UAuthority> = uprotocol::v1::UAuthority(),
                std::optional<uprotocol::v1::UEntity> = uprotocol::v1::UEntity()) noexcept;
```

*OR*

```suggestion
            static std::shared_ptr<UpZenohClient> instance(
                std::optional<uprotocol::v1::UAuthority> = std::nullopt_t,
                std::optional<uprotocol::v1::UEntity> = std::nullopt_t) noexcept;
```